.travis.yml

```diff
@@ -69,7 +80,16 @@ script:
 
 after_success:
  - ccache -s
- - if [[ ${COVERAGE} ]]; then pushd build && bash <(curl -s https://codecov.io/bash) && popd; fi
+ - test "${TRAVIS_BUILD_STAGE_NAME}" = "Test" || travis_terminate 0
+ - if [[ ${COVERAGE} ]]; then
+     pushd build && wget -O codecov https://codecov.io/bash && chmod +x codecov &&
+     if [[ ${CC} = clang ]]; then
+       ./codecov -x "llvm-cov gcov" -F "${CC}";
+     else
+       ./codecov -F "${CC}";
+     fi &&
```

Comment:
You did not advertise you were changing logic to upload the code coverage in this PR.

Reply:
Done!